runtime/src/accounts_db.rs

```diff
@@ -4230,6 +4230,23 @@ impl AccountsDb {
         result
     }
 
+    fn sort_storages(storages: &[SnapshotStorage]) -> Vec<&Vec<Arc<AccountStorageEntry>>> {
```

Comment:
[Naming] Since Rust sorts in place, I think this function is surprising at first glance since it does not sort in place. In C++, many of the algorithms that make a copy are appended with `_copy`. Rust has `into` and `from` as a common thing I've seen, but I dunno if they always make copies...

Anyway, maybe something like `sort_storages_copy()`?